akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraOffsetStore.scala

```diff
@@ -29,32 +29,45 @@ import akka.stream.alpakka.cassandra.scaladsl.CassandraSession
   // FIXME make keyspace and table names configurable
   val keyspace = "akka_projection"
   val table = "offset_store"
+  val cassandraPartitions = 5
+
+  private[cassandra] val partitionClause = (0 until cassandraPartitions).mkString(",")
 
   def this(session: CassandraSession)(implicit ec: ExecutionContext) =
     this(session, Clock.systemUTC())
 
   def readOffset[Offset](projectionId: ProjectionId): Future[Option[Offset]] = {
     session
-      .selectOne(s"SELECT offset, manifest FROM $keyspace.$table WHERE projection_id = ?", projectionId.id)
-      .map { maybeRow =>
-        maybeRow.map(row => fromStorageRepresentation[Offset](row.getString("offset"), row.getString("manifest")))
+      .selectAll(
+        s"SELECT projection_key, offset, manifest FROM $keyspace.$table WHERE projection_name = ? AND partition IN ($partitionClause)",
```

Comment:
We know the `partition` here, so use that instead of query all of them.
Same partition from projectionId.key as when writing.

Reply:
and it should include `projection_key` in the WHERE, so it's 
```
projection_name = ? AND partition = ? AND projection_key = ?
```

Reply:
I wrote it this way in anticipation of supporting `MergeableOffset`, so that there wouldn't need to be several query implementations.  I understand it would optimize the read use case for all other offsets, but wasn't sure if it would be worth the minor performance improvement.

Reply:
What are the implications of always querying on all partitions? 

In any case, if we support `MergeableOffset`, we will need this, IIUC.

Reply:
Let's implement the MergeableOffset when we implement, and start here with the simplest and most efficient query.

Reply:
May I suggest that we bring #97 into MVP scope and I'll implement both read use cases in this PR?  I can finish #97 fairly quickly in a follow up.

Reply:
We have other more important things to complete first. For example the Kafka exactly-once problem is more important.

Reply:
Alright.  To move this forward I'll only implement the simple use case.

Are you referring to #98?  Should we bring that into MVP?

Reply:
Yes, that is absolutely MVP, otherwise we can’t support Kafka for exactly-once, which was the whole purpose of the Kafka source. Sorry, i must have missed assigning it to the MVP board.

Reply:
Ok.  I moved it to MVP.

I've implemented the simple read use case as suggested.